js/app/sai.js

```diff
@@ -306,6 +308,22 @@
       } else {
         $scope.SAIScript += "DELETE FROM `smart_scripts` WHERE (`source_type` = " + $stateParams.sourceType + " AND `entryorguid` = " + $stateParams.entryOrGuid + ");\n";
       }
+      
+
+      
+      $http({
+          method  : 'POST',
+          url     : 'http://localhost/dev/sai_parser.php',
```

Comment:
this path should be a relative path not an [absolute path](https://en.wikipedia.org/wiki/Path_(computing)#Absolute_and_relative_paths)